### PR TITLE
friends-app

### PR DESCRIPTION
**Pull request by sayckl**
### friends-app

[Demo](https://sayckl.github.io/friends-app/) |
[Code base](https://github.com/sayckl/frontend-2019-p2p/tree/friends-app-task/submissions/sayckl/friends-app)

The code is submitted in its own sub-directory and in a dedicated feature branch.
Please, review.

### CODE REVIEW

File: submissions/sayckl/friends-app/script.js [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+  });
+};
+
+function capitals(elem) {
+  elem.name.first = capitalFirstLet(elem.name.first);
+  elem.name.last = capitalFirstLet(elem.name.last);
+  elem.location.city = capitalFirstLet(elem.location.city);
+}
+
+function capitalFirstLet(word) {
```
Comment (zonzujiro) [inline submissions/sayckl/friends-app/script.js]: You can use css instead

File: submissions/sayckl/friends-app/script.js [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+  return word.charAt(0).toUpperCase() + word.slice(1);
+}
+
+az.addEventListener('click', () => {
```
Comment (zonzujiro) [inline submissions/sayckl/friends-app/script.js]: `az` is a bad naming - it is very unobvious. Please, find the right name

File: submissions/sayckl/friends-app/script.js [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+    const ads = people.sort((a, b) => {
```
Comment (zonzujiro) [inline submissions/sayckl/friends-app/script.js]: Same with naming. What is `ads`?

File: submissions/sayckl/friends-app/script.js [hunk 4]
```diff
@@ -0,0 +1,115 @@
...
+
+reset.addEventListener('click', () => {
+  cards.innerHTML = '';
+  add(people);
+  searcher.value = '';
+});
+
+const searcher = document.querySelector('.searcher');
+function search(people) {
+let search = new RegExp(searcher.value, "ig");
```
Comment (zonzujiro) [inline submissions/sayckl/friends-app/script.js]: Regexp can be replaced with a string method.
Also, you missed spaces inside functions at the beginning of line. You must use prettier for such stuff

File: submissions/sayckl/friends-app/style.css [hunk 5]
```diff
@@ -0,0 +1,96 @@
...
+.menu-btn_active span:after {
+  transform: rotate(-45deg);
+}
+body  {
+  margin: 0;
+}
+.body {
+
+}
+aside {
```
Comment (zonzujiro) [inline submissions/sayckl/friends-app/style.css]: Styles on tags is a bad idea - they are too general. Give a class name

File: submissions/sayckl/friends-app/script.js [hunk 6]
```diff
@@ -0,0 +1,115 @@
...
+     add(staticPeople);
+   });
+
+function add(people){
+  people.forEach(function(elem) {
+    capitals(elem);
+    const div = document.createElement('div');
+    div.className = 'card';
+    div.innerHTML = `<img src="${elem.picture.large}">
+    <div class="name">${elem.name.first} ${elem.name.last}</div>
```
Comment (zonzujiro) [inline submissions/sayckl/friends-app/script.js]: Maybe `span` of `p` will fit better? Like for text element

File: submissions/sayckl/friends-app/script.js (hunk 1 above)
Comment (sayckl) [reply submissions/sayckl/friends-app/script.js]: done, used text-transform: capitalize

File: submissions/sayckl/friends-app/script.js (hunk 2 above)
Comment (sayckl) [reply submissions/sayckl/friends-app/script.js]: changed to textASC
maybe better names/fullname ASC

File: submissions/sayckl/friends-app/script.js (hunk 3 above)
Comment (sayckl) [reply submissions/sayckl/friends-app/script.js]: it should be asc*, like ASCending, my fail

File: submissions/sayckl/friends-app/style.css (hunk 5 above)
Comment (sayckl) [reply submissions/sayckl/friends-app/style.css]: aside added class

File: submissions/sayckl/friends-app/script.js (hunk 6 above)
Comment (sayckl) [reply submissions/sayckl/friends-app/script.js]: changed div's to span's

File: submissions/sayckl/friends-app/script.js (hunk 4 above)
Comment (sayckl) [reply submissions/sayckl/friends-app/script.js]: I'll try

File: submissions/sayckl/friends-app/script.js (hunk 3 above)
Comment (zonzujiro) [reply submissions/sayckl/friends-app/script.js]: `ascending` is better

File: submissions/sayckl/friends-app/script.js (hunk 2 above)
Comment (zonzujiro) [reply submissions/sayckl/friends-app/script.js]: https://github.com/kottans/frontend-2019-p2p/pull/77#discussion_r296151214

and with `ascending`. Better use full words in names of variables

**Comment by zonzujiro** [comment]
Pls, resolve outdated discussions

File: submissions/sayckl/friends-app/script.js (hunk 3 above)
Comment (sayckl) [reply submissions/sayckl/friends-app/script.js]: 👍

**Pull request merged**